src/ACadSharp/Entities/Spline.cs

```diff
@@ -112,7 +114,82 @@ public Spline() : base() { }
 		/// <inheritdoc/>
 		public override BoundingBox GetBoundingBox()
 		{
-			return BoundingBox.FromPoints(this.ControlPoints);
+			List<XYZ> vertices = this.PolygonalVertexes(256);
+
+			return BoundingBox.FromPoints(vertices);
+
+			// previous implementation: control points can be outside the bounding box of the spline curve
+			//return BoundingBox.FromPoints(this.ControlPoints);
+		}
+
+		/// <summary>
+		/// Converts the spline in a list of vertexes.
+		/// </summary>
+		/// <param name="precision">Number of vertexes generated.</param>
+		/// <returns>A list vertexes that represents the spline expressed in object coordinate system.</returns>
+		public List<XYZ> PolygonalVertexes(int precision)
```

Comment:
netDxf has an equivalent method in the archive:
https://github.com/haplokuon/netDxf/blob/3d24d7c23418bd3a077d0e32a639f2ffe95bff51/netDxf/Entities/Spline.cs#L535

Can we add a test for this method? just to be sure that we are checking the correct functionality.